### PR TITLE
Add Shingeki no Shoujo

### LINKED ISSUE #10903: Shingeki no Shoujo
### Source name

Shingeki no Shoujo

### Source link

https://shingekinoshoujo.to/

### Language

Italian

### Other details

Shingeki no Shoujo is an italian team that translates manga, manhwa, novel.
I have their consent to request for an extension.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a title with source name.
- [X] I have checked that the extension does not already exist on the [website extensions list](https://tachiyomi.org/extensions/) or the app.
- [X] I have checked that the extension does not already exist by searching the [GitHub repository](https://github.com/tachiyomiorg/tachiyomi-extensions/) and verified it does not appear in the code base.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by ReLoia**
Closes #10903

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by arkon** [comment]
The site seems like some sort of Wordpress theme. I'm not sure if one of the existing multisrc generators would work for this too?

**Comment by AntsyLich** [comment]
Doesn't seem like there is an existing extension with "Flash" theme

**Comment by ReLoia** [comment]
Yes, I haven't found any multisrc generator that seems to be related to Flash theme. 
[That theme](https://it.wordpress.org/themes/flash/) doesn't even seem to be related to manga/comics.

File: src/it/shingekinoshoujo/build.gradle [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
```
Comment (arkon) [inline src/it/shingekinoshoujo/build.gradle:3]: Don't need this.

**Pull request merged**